target/linux/ramips/mt7620/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
New files should have an SPDX header.

Please don't copy this stone-age version, but take a recent one if we really need it.

Reply:
Refactored with new "common code" version